packages/editor/src/client/blocks/teaserFlexGridBlock.tsx

```diff
@@ -0,0 +1,278 @@
+import React, {useEffect, useState} from 'react'
+import GridLayout from 'react-grid-layout'
+
+import 'react-grid-layout/css/styles.css'
+import 'react-resizable/css/styles.css'
+
+import {FlexItemAlignment, FlexTeaser, Teaser, TeaserFlexGridBlockValue} from './types'
+import {BlockProps} from '../atoms/blockList'
+import nanoid from 'nanoid'
+import {ButtonToolbar, Drawer, Icon, IconButton, Panel} from 'rsuite'
+import {IconButtonTooltip} from '../atoms/iconButtonTooltip'
+import {contentForTeaser, TeaserBlockProps} from './teaserGridBlock'
+import {PlaceholderInput} from '../atoms/placeholderInput'
+import {TeaserEditPanel} from '../panel/teaserEditPanel'
+import {TeaserSelectAndEditPanel} from '../panel/teaserSelectAndEditPanel'
+
+// import {useTranslation} from 'react-i18next'
```

Comment:
can you clean this one?

Reply:
@bishoymelek the translation import? I'll use it.